packages/core/src/physics/shape/ColliderShape.ts

```diff
@@ -133,6 +135,27 @@ export abstract class ColliderShape implements ICustomClone {
     Engine._physicalObjectsMap[this._id] = this;
   }
 
+  /**
+   * Get the distance and the closest point on the shape from a point.
+   * @param point - The point
+   * @param outClosestPoint - The result of the closest point on the shape
+   * @returns The distance between the point and the shape
+   */
+  getDistanceAndClosestPointFromPoint(point: Vector3, outClosestPoint: Vector3): number {
+    const tempQuat = ColliderShape._tempWorldRot;
```

Comment:
should return input point and distance 0 if collider is not active

Reply:
done